src/commands/root.js

```diff
@@ -0,0 +1,79 @@
+const handleErrors = require("../utils/handleErrors");
+const fs = require("fs");
+const copy = require("recursive-copy");
+const {
+  getProdURI
+} = require("../../templates/infra-authentication-server/src/utils/getConfigFile");
+const User = require("../../templates/infra-authentication-server/src/models/User");
+const mongoose = require("mongoose");
+
+module.exports.command = "create-root [email]";
+module.exports.describe =
+  "creates root user with [email] and returns passwword";
+module.exports.builder = yargs => yargs;
+
+module.exports.handler = handleErrors(async argv => {
+  // spin up the prod db
+  // getUsers
+  const uri = await getProdURI();
+
+  if (!uri) {
+    console.error("Unable to get uri from config yaml file");
+    return;
+  }
+  await mongoose.connect(uri, { useNewUrlParser: true });
```

Comment:
Can we wrap this in a try catch, and have it return an error when mongoose fails to connect